charts/iaa-device-plugin/templates/iaa.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
   annotations: {{ toYaml .Values.annotations | nindent 4 }}
 spec:
   image: "{{ .Values.image.hub }}/intel-iaa-plugin:{{ .Values.image.tag | default .Chart.AppVersion }}"  
-  initImage: "{{ .Values.initImage.hub }}/intel-iaa-plugin:{{ .Values.initImage.tag | default .Chart.AppVersion }}" 
+  initImage: "{{ .Values.initImage.hub }}/intel-iaa-initcontainer:{{ .Values.initImage.tag | default .Chart.AppVersion }}" 
```

Comment:
```suggestion
  initImage: "{{ .Values.initImage.hub }}/intel-idxd-config-initcontainer:{{ .Values.initImage.tag | default .Chart.AppVersion }}" 
```

Reply:
Is the same change required for DLB?

Reply:
no, just DSA and IAA belong to this "IDXD" device category

Reply:
`initImage` | `intel/intel-iaa-initcontainer:0.25.0` of README.md should be also fixed?

Reply:
for `image` and `initImage` we only have `hub` and `tag` so yes these should all be revisited

Reply:
done